platform/darwin/src/MGLStyle.h

```diff
@@ -442,6 +442,22 @@ MGL_EXPORT
  */
 - (void)removeImageForName:(NSString *)name;
 
+#pragma mark Managing a Style’s Transition Options
+
+/**
+ The duration in seconds to animate any changes to the style URL or to layout and paint attributes.
```

Comment:
Huh, I overlooked the fact that changes to the style URL can also be animated. That’s kind of awkward, since changing the style URL means swapping one MGLStyle object for another.